app/models/concerns/extended_cache_key.rb

```diff
@@ -32,7 +32,7 @@ def cache_key
     cache_key = super
     associations_cache_key = compound_association_cache_keys.join("+")
     methods_cache_key = compound_method_cache_keys.join("")
-    create_append_cache_key(cache_key, methods_cache_key, associations_cache_key)
+    create_append_cache_key(+cache_key, methods_cache_key, associations_cache_key)
```

Comment:
TIL about this syntax - nice! 
```suggestion
    # dup the frozen string https://ruby-doc.org/core-2.6/String.html#method-i-2B-40
    create_append_cache_key(+cache_key, methods_cache_key, associations_cache_key)
```

FWIW - this is the sort of link / comment that ideally would be included in the commit message as well, https://github.com/zooniverse/panoptes/pull/3998/commits/d52f664109668b19c4e69c57b078ec8cce596554